org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2.yml

```diff
@@ -0,0 +1,31 @@
+id: org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2
+branch: '22.08'
+build-extension: true
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-options:
+  prefix: /app/extensions/Plugins/NeuralAmpModelerLV2
+modules:
+  - name: NeuralAmpModelerLV2
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - "mv ${FLATPAK_DEST}/lib/lv2 ${FLATPAK_DEST}"
+      - "install -Dm644 LICENCE.md ${FLATPAK_DEST}/share/licenses/NeuralAmpModelerLV2/LICENSE"
+      - "install -Dm644 deps/NeuralAmpModelerCore/LICENSE ${FLATPAK_DEST}/share/licenses/NeuralAmpModeler/LICENSE"
+      - "install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2.metainfo.xml"
+      - "appstream-compose --basename=org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2 --prefix=${FLATPAK_DEST} --origin=flatpak org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2"
+    cleanup:
+      - lib
+    sources:
+      - type: git
+        url: https://github.com/mikeoliphant/neural-amp-modeler-lv2
+        branch: main
```

Comment:
```suggestion
```

never a branch

Reply:
this is the cause of the build failure.

Reply:
Right, that was an oversight. Thanks for spotting it. <3